_includes/scripts.html

```diff
@@ -41,46 +41,55 @@
     a.appendChild(r);
   })(window, document, "https://static.hotjar.com/c/hotjar-", ".js?sv=");
 </script>
+{% endif %}
+
+<!-- Intercom -->
 <script>
   window.intercomSettings = {
     app_id: "rzcjacgd"
   };
 </script>
-{% endif %}
-
-<!-- Intercom -->
 <script>
-  (function() {
+  (function () {
     var w = window;
     var ic = w.Intercom;
     if (typeof ic === "function") {
       ic("reattach_activator");
-      ic("update", intercomSettings);
+      ic("update", w.intercomSettings);
     } else {
       var d = document;
-      var i = function() {
+      var i = function () {
         i.c(arguments);
       };
       i.q = [];
-      i.c = function(args) {
+      i.c = function (args) {
         i.q.push(args);
       };
       w.Intercom = i;
-      function l() {
+      var l = function () {
         var s = d.createElement("script");
         s.type = "text/javascript";
         s.async = true;
-        s.src = "https://widget.intercom.io/widget/rzcjacgd";
+        s.src = `https://widget.intercom.io/widget/${w.intercomSettings.app_id}` ;
         var x = d.getElementsByTagName("script")[0];
         x.parentNode.insertBefore(s, x);
-      }
-      if (w.attachEvent) {
+      };
+      if (document.readyState === "complete") {
+        l();
+      } else if (w.attachEvent) {
         w.attachEvent("onload", l);
       } else {
         w.addEventListener("load", l, false);
       }
     }
-  })();
+})();
 </script>
+<script>
+  window.Intercom("boot", {
+    app_id: window.intercomSettings.app_id
+  });
+</script>
```

Comment:
What's this for?

Reply:
To start up the intercom messenger. https://developers.intercom.com/installing-intercom/docs/basic-javascript#section-step-2-launch-messenger